pkg/motor/x/logger/logger.go

```diff
@@ -0,0 +1,64 @@
+package logger
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+)
+
+type LogLevel uint32
+
+const (
+	LEVEL_INFO  LogLevel = 0
+	LEVEL_DEBUG LogLevel = 1
+	LEVEL_WARN  LogLevel = 2
+	LEVEL_ERROR LogLevel = 3
+	LEVEL_FATAL LogLevel = 4
+)
+
+const (
+	CAT_INFO  = "[INFO]"
+	CAT_DEBUG = "[DEBUG]"
+	CAT_WARN  = "[WARN]"
+	CAT_ERROR = "[ERROR]"
+	CAT_FATAL = "[FATAL]"
+)
+
+type Logger struct {
+	logger   *log.Logger
+	level    LogLevel
+	category string
+}
+
+func New(level string, category string) *Logger {
```

Comment:
It looks like the `category` isn't actually used anywhere. Does it need to be part of the struct?